PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -5,6 +5,18 @@ namespace PrincessBrideTrivia.Tests;
 [TestClass]
 public class ProgramTests
 {
+    //(x,y,z) = correct, hints, adjusted answer; hints should never exceed correct
+    [TestMethod]
+    [DataRow(7,1,6.5)]
+    [DataRow(6,2,5)]
+    [DataRow(5,3,3.5)]
+    [DataRow(4,4,2)]
+    public void AdjustHints_AdjustsNumberCorrect(int numCorrect, int numHints, double expected)
```

Comment:
```suggestion
    public void AdjustHints_GivenCorrectAnswersAndHintsGiven_AdjustsFinalScore(int numCorrect, int numHints, double expected)
```
Just a suggestion for potentially a cleaner name